### PR TITLE
feat: womprotocol

### PR DESCRIPTION
**Pull request by krishguptadev**
#### Checklist

**I agree to the following :-**

*   Added description of the change
*   I've read the [contributing guidelines](https://github.com/web3community/devprotocol.xyz/blob/main/CONTRIBUTING.md)
*   Search previous suggestions before making a new PR, as yours may be a duplicate.
*   I acknowledge that all my contributions will be made under the project's license.

Notes (**optional**): 

<a href="https://gitpod.io/#https://github.com/web3community/devprotocol.xyz/pull/157"></a>

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
Let's wait for the other maintainers to review. :)

**Review by vinzvinci — COMMENTED** [review]
This is not done, please don't approve. The following information needed is in the discord collaborators channel.

File: src/pages/index.astro
Comment (krishguptadev) [inline src/pages/index.astro:36]:
```suggestion
  <section class="flex pb-28 md:py-8 flex-wrap items-center justify-around">
```

**Comment by vinzvinci** [comment]
I don't know if this integration will continue, I'll ask Mayu and Pdot on the discord server.

**Comment by vinzvinci** [comment]
It will continue. Let's wait for P. direction as Mayu said

**Comment by krishguptadev** [comment]
> It will continue. Let's wait for P. direction as Mayu said

I think aggresan approved this 😅

**Review by vinzvinci — DISMISSED** [review]
LGTM 👍 Ready to merge! Check discord message

**Review by shuklaritvik06 — APPROVED** [review]
Lgtm

**Review by vinzvinci — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**
